package.json

```diff
@@ -88,8 +88,8 @@
   },
   "peerDependencies": {
     "big.js": "^6.2.2",
-    "nhsuk-frontend": ">=8.0.0 <9.0.0",
-    "nhsuk-react-components": ">=4.0.0 <5.0.0",
+    "nhsuk-frontend": ">=8.0.0 <=10.0.0",
+    "nhsuk-react-components": ">=4.0.0 <=6.0.0",
```

Comment:
I don't think we can confidently say this will work with versions 10 & 6 so we don't want to include them

Reply:
resolved